nav_msgs/Path target_path
float32 speed
